Quadriga/src/main/java/edu/asu/spring/quadriga/domain/factory/impl/workbench/PublicPageFactory.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.spring.quadriga.domain.factory.impl.workbench;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.factory.workbench.IPublicPageFactory;
+import edu.asu.spring.quadriga.domain.impl.workbench.PublicPage;
+import edu.asu.spring.quadriga.domain.impl.workbench.PublicStatisticsPage;
+import edu.asu.spring.quadriga.domain.workbench.IPublicPage;
+
+/**
+ * Factory class to create Public Page object
+ * 
+ * @author Prasanth Priya Nesan
+ * 
+ */
+@Service
+public class PublicPageFactory implements IPublicPageFactory {

```

Comment:
My thinking was since it's a part of arranging settings for public page and since we have public page factory - I thought they are linked, as if using factory pattern.

But, okay I will do it the way which you described.